### PR TITLE
Update configure-the-outbound-spam-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about the NDR's that get generated when automatic forwarding is disabled

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3284

File: microsoft-365/security/office-365-security/configure-the-outbound-spam-policy.md [hunk 1]
```diff
@@ -172,6 +172,9 @@ Creating a custom outbound spam policy in the Security & Compliance Center creat
 
    - **Off**: All automatic external email forwarding is disabled by the policy.
 
+ > [!NOTE]
+ > When Automatic Forwarding is disabled, For external senders sending to a mailbox that has forwarding in place, The recipient will get an NDR that they are not allowed to forward. For internal senders the sender will get the NDR.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-the-outbound-spam-policy.md]:
```suggestion
 > When Automatic Forwarding is disabled, the recipient will receive a non-delivery report (NDR) if external senders send email to a mailbox that has forwarding in place. If the email is sent by an internal sender, the sender will get the NDR.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrisda Please merge this PR. Thanks.

**Pull request merged**